[Plugin] Renew password periodically or other

yebor974 opened the pull request:
Add new plugi to renew password periodically or according to other criteria.

saade reviewed (changes requested):
Hey! Could you please ensure that image URLs in your plugin's `README` file are absolute URLs rather than relative links? Then we can embed them on our website correctly. Thanks!

Please let us know when you've addressed these issues.

yebor974 commented:
> Hey! Could you please ensure that image URLs in your plugin's `README` file are absolute URLs rather than relative links? Then we can embed them on our website correctly. Thanks!
> 
> Please let us know when you've addressed these issues.

@saade it's done, ty
